shared/src/components/message.tsx

```diff
@@ -31,42 +31,41 @@ interface Props {
 const localStyles = stylesheet({
   error: {
     backgroundColor: 'var(--md-red-50, #ffebee)',
```

Comment:
I'm wondering if there's a better background color based on the CSS variables at https://github.com/jupyterlab/jupyterlab/blob/1.2.x/packages/theme-light-extension/style/variables.css?

Reply:
http://b/175132231

Reply:
Changed to  --jp-error-color3: var(--md-red-100);